testing/fs-uae-launcher/APKBUILD

```diff
@@ -1,20 +1,18 @@
 # Maintainer: Taner Tas <taner76@gmail.com>
 pkgname=fs-uae-launcher
-pkgver=2.8.4
+pkgver=3.0.0
 pkgrel=0
 pkgdesc="FS-UAE emulator launcher."
 url="https://fs-uae.net/launcher"
 arch="noarch !armhf !armv7 !aarch64 !s390x"  # fs-uae not avail
 license="GPL-2.0-only"
-depends="fs-uae~$pkgver py3-qt5"
+depends="fs-uae=$pkgver py3-qt5 py3-sip-pyqt5"
```

Comment:
Err...sorry, should be `~$pkgver` not `=$pkgver`, my bad. Seems to work for me then.

Reply:
I did it with updated commit message.